xml/System.Data.Spatial/DbGeography.xml

```diff
@@ -881,8 +881,8 @@
         <ReturnType>System.Nullable&lt;System.Double&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets the Longitude coordinate of this DbGeography value, if it represents a point. &lt;returns&gt;The Longitude coordinate value of this geography value, if it represents a point; otherwise <see langword="null" />.&lt;/returns&gt;</summary>
-        <value>The Longitude coordinate of this DbGeography value.</value>
+        <summary>Gets the Longitude coordinate of this DbGeography value, if it represents a point.</summary>
+        <value>The Longitude coordinate value of this geography value, if it represents a point; otherwise <see langword="null" />.</value>
```

Comment:
```suggestion
        <value>The Longitude coordinate value of this geography value, if it represents a point; otherwise, <see langword="null" />.</value>
```